### PR TITLE
14 lakh changes

### PR DESCRIPTION
**Pull request by snopoke**
review by commit

### CODE REVIEW

File: configs/icds-14lakh.yml [hunk 1]
```diff
@@ -264,11 +263,11 @@ services:
...
           unit_size: 683  # casetimedscheduleinstance
     process:
-      cores_per_node: 16
-      ram_per_node: 48
+      cores_per_node: 32
+      ram_per_node: 128
 
   pg_proxy:
-    usage_capacity_per_node: 250000
+    usage_capacity_per_node: 300000
```
Comment (snopoke) [inline configs/icds-14lakh.yml:271]: @NitigyaS is it ok to change the usage capacity without increasing cores?

File: configs/icds-14lakh.yml [hunk 2]
```diff
@@ -406,7 +405,7 @@ services:
       ram_static_baseline: 1  # per node
 
   pg_ucr:
-    usage_capacity_per_node: 50000
+    usage_capacity_per_node: 100000
```
Comment (snopoke) [inline configs/icds-14lakh.yml:409]: this is also limited by the disk performance which isn't really captures in this tool at all. If we double the usage we'd need to double to disk performance as well (until we have citus rolled out)

**Comment by snopoke** [comment]
@NitigyaS I'm going to merge this so I can reference it in a doc

File: configs/icds-14lakh.yml [hunk 3]
```diff
@@ -0,0 +1,563 @@
...
+      cores_per_node: 32
+      ram_per_node: 128
+
+  couchdb:
+    usage_capacity_per_node: 100000
+    min_nodes: 3
+    storage:
+      group: 'SSD'
+      redundancy_factor: 3
+      static_baseline: 20GB  # to account for other databases
```
Comment (emord) [inline configs/icds-14lakh.yml:319]: Does this also account for the need for extra disk space when compacting?

File: configs/icds-14lakh.yml [hunk 4]
```diff
@@ -0,0 +1,563 @@
...
+    storage:
+      group: 'VM_other'
+      static_baseline: 50GB
+      add_buffer: False
+      override_estimation_buffer: 0
+    process:
+      cores_per_node: 4
+      ram_per_node: 8
+
+  kafka:
```
Comment (emord) [inline configs/icds-14lakh.yml:360]: There's a comment on some of the other services that the limits aren't well known. Do you think that applies here? What have you found to be the limiting factor?

File: configs/icds-14lakh.yml (hunk 3 above)
Comment (snopoke) [reply configs/icds-14lakh.yml:319]: Probably not, thanks for flagging. I'll override the buffer for couch to make it bigger. Any thoughts on how how much to account for (%)?

File: configs/icds-14lakh.yml (hunk 4 above)
Comment (snopoke) [reply configs/icds-14lakh.yml:360]: that probably applies to kafka as well but it's much easier to scale up whereas things like RabbitMQ don't scale horizontally so it's not as easy.

File: configs/icds-14lakh.yml (hunk 3 above)
Comment (snopoke) [reply configs/icds-14lakh.yml:319]: Needs free space equal to the size of the view / DB being compacted

**Pull request merged**